cypress/integration/ui/creatives.js

```diff
@@ -0,0 +1,136 @@
+const { clone, extend } = require('lodash');
+const { login } = require('../../helpers/login-helper');
+const nameHelper = require('../../helpers/name-helper');
+
+const getRequest = (options = {}) => {
+  const defaultOptions = {
+    auth: {
+      bearer: Cypress.env('authToken'),
+    },
+    url: '/api/v1/creative',
+  };
+  return extend(defaultOptions, options);
+};
+
+context('Creatives', () => {
+  describe('Creatives UI', () => {
+    let userSessionToken;
+    let creativeID;
+    let snippetGenerated;
+    const creativeHTML = nameHelper.generateName('generated_creative');
+
+
+    before(() => {
+      userSessionToken = login('kargoqa@gmail.com', 'K@rgo123!');
+    });
+
+    // This beforeEach is needed because when a subsequent tests hits
+    // the same URL, cookies is not properly retained
+    beforeEach(() => {
+      cy.setCookie('kauth_access', userSessionToken);
+    });
+
+    const dealGroupName = nameHelper.generateName('API_automated-Deal_Group_created');
+    const creativeName = nameHelper.generateName('UI_automated-Creative_created');
+    let dealGroupId;
+
+    it('API-Add a deal group', () => {
+      cy.fixture('deal-group.json').then((dealgroup) => {
+        const reqBody = clone(dealgroup);
+        reqBody.name = dealGroupName;
+        reqBody.salesforce_id = Cypress.moment().format('[ui-]YYYYMMMDDhhmmss'); // adds Unique Salesforce ID
+        reqBody.is_test = false;
+
+        const requestOptions = getRequest({
+          method: 'POST',
+          body: reqBody,
+          url: '/api/v1/deal-group',
+        });
+
+        cy.request(requestOptions)
+          .then((resp) => {
+            dealGroupId = resp.body.id;
+            assert.equal(resp.status, 200, 'status');
+            assert.isAbove(resp.body.id, 0, 'Deal-group ID ');
+            cy.log('Created Deal-Group via API');
+          });
+      });
+    });
```

Comment:
This should find an existing deal group and use it.   Make this change if you feel like you have time..